[Hyperdrive] Improve tutorial (#11982)
cloudflare/cloudflare-docs: content/hyperdrive/tutorials/serverless-timeseries-api-with-timescale/index.md

* `Do you want to use TypeScript?`: Select `Yes`.
* `Do you want to deploy your application?`: Select `Yes`.

Make note of the URL that your application was deployed to. You will be using it when you configure your GitHub webhook.

Change into the directory you just created for your Worker project:

```sh
$ cd timescale-api
```

## 2. Prepare your Timescale Service

{{<Aside type="note">}}
If you have not signed up for Timescale, go to the [signup page](https://timescale.com/signup) where you can start a free 30 day trial with no credit card.
{{</Aside>}}

If you are creating a new service, go to the [Timescale Console](https://console.cloud.timescale.com/) and follow these steps:

1. Select **Create Service** by selecting the black plus in the upper right.
2. Choose **Time Series** as the service type. 
3. Choose your desired region and instance size. 1 CPU will be enough for this tutorial.
4. Set a service name to replace the randomly generated one.
5. Select **Create Service**.
6. On the right hand side, expand the **Connection Info** dialog and copy the **Service URL**.
7. Copy the password which is displayed. You will not be able to retrieve this again.

> 8. Select 'I stored my password, go to service overview'.

Reviewer: Should this also be in bold, since it seems to be a UI element?

## 3. Create your Hypertable

Timescale allows you to convert regular PostgreSQL tables into [hypertables](https://docs.timescale.com/use-timescale/latest/hypertables/), tables used to deal with time-series, events, or analytics data. Once you have made this change, Timescale will seamlessly manage the hypertable's partitioning, as well as allow you to apply other features like compression or continuous aggregates.

Connect to your Timescale database using the Service URL you copied in the last step (it has the password embedded). 

If you are using the default PostgreSQL CLI tool [**psql**](https://www.timescale.com/blog/how-to-install-psql-on-mac-ubuntu-debian-windows/) to connect, you would run psql like below (substituting your **Service URL** from the previous step). You could also connect using a graphical tool like [PgAdmin](https://www.pgadmin.org/).

> ```bash

Reviewer: ```suggestion
```sh
```

> $ psql SERVICEURL

Reviewer: `SERVICEURL` seems to be a placeholder... Right?
If that's the case, I'd use `<SERVICEURL>` — this would imply other changes below.

{{</Aside>}}

> This command outputs your Hyperdrive ID.  You can now bind your Hyperdrive configuration to your Worker in your `wrangler.toml` configuration by replacing the content with the following:

Reviewer: ```suggestion
This command outputs your Hyperdrive ID. You can now bind your Hyperdrive configuration to your Worker in your `wrangler.toml` configuration by replacing the content with the following:
```

```

Now copy the below Worker code, and replace the current code in `./src/index.ts`.  The code below:

1. Uses Hyperdrive to connect to Timescale using the connection string generated from `env.HYPERDRIVE.connectionString` directly to the driver.
2. Creates a `POST` route which accepts an array of JSON readings to insert into Timescale in one transaction.

> 3. Creates a `GET` route which will take limit parameter and then return the most recent readings. This could be adapted to filter by ID or by timestamp.

Reviewer: ```suggestion
3. Creates a `GET` route which takes a `limit` parameter and returns the most recent readings. This could be adapted to filter by ID or by timestamp.
```

## 5. Deploy your Worker

Run the following command to redeploy your Worker:

> ```bash

Reviewer: ```suggestion
```sh
```
